test/cluster/object_store/test_basic.py

```diff
@@ -261,8 +262,30 @@ async def test_get_object_store_endpoints(manager: ManagerClient, s3_server):
     print('Check that system.config contains the object storage endpoints')
     cql = manager.get_cql()
     res = json.loads(cql.execute("SELECT value FROM system.config WHERE name = 'object_storage_endpoints';").one().value)
-    print(res)
-    print(objconf)
     assert s3_server.address in res and 'a' in res
     assert json.loads(res[s3_server.address]) == objconf[0]
     assert json.loads(res['a']) == badconf[0]
+
+    print('Update config with a new endpoint and SIGHUP Scylla to reload configuration')
+    new_endpoint = MinioServer.create_conf('b', 456, 'good_region')
+    await manager.server_update_config(server.server_id, 'object_storage_endpoints', new_endpoint)
+    await wait_for_config(manager, server, 'object_storage_endpoints', {'b': '{ "port": 456, "use_https": false, "aws_region": "good_region", "iam_role_arn": "" }'})
```

Comment:
Why is `"iam_role_arn": ""` part here? It looks like config creation doesn't set this field, so why do you wait for it to appear?

Reply:
I wait for `object_storage_endpoints` to appear, `iam_role_arn` is returned with the default value by the API.